mirrord/config/src/target.rs

```diff
@@ -252,6 +252,7 @@ pub enum Target {
 
     /// <!--${internal}-->
     /// Spawn a new pod.
+    #[default]
```

Comment:
This looks kind of error prone tbh, I wouldn't add `Default` here

Reply:
Removed